src/slices/filtersSlice.js

```diff
@@ -0,0 +1,28 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const filtersSlice = createSlice({
+  name: 'filters',
+  initialState: { skills: [], projects: [] },
+  reducers: {
+    setSkillsFilter(state, action) {
+      return { ...state, skills: [ ...action.payload ] };
+    },
+    resetSkillsFilter(state) {
```

Comment:
As an alternative you could define a function that would just call `setSkillsFilter` with an empty array, such as

```js
const resetSkillsFilter = () => setSkillsFilter([]);
```